modules/container-insights/variables.tf

```diff
@@ -1,13 +1,16 @@
-variable "enabled" {
-  description = "A conditional indicator to enable cluster-autoscale"
-  type        = bool
-  default     = true
+variable "features" {
+  description = "Toggle flags to enable cloudwatch features"
+  type        = map(any)
+  default = {
+    enable_metrics = false
+    enable_logs    = false
+  }
```

Comment:
Changing the enabled var from a boolean to a map is a breaking change that should come with a major version bump.
but I am ok with that because I think nobody is using enabled variable right now.

Reply:
Agreed. major version upgrade is required.

Reply:
Yes, this is a breaking change. I think, new version will go up to `1.6.0`.